src/test/java/org/mitre/synthea/world/agents/PayerTest.java

```diff
@@ -54,163 +56,237 @@ public static void setup() throws Exception {
     testState = Config.get("test_state.default", "Massachusetts");
     // Set up Medicaid numbers.
     healthInsuranceModule = new HealthInsuranceModule();
-    double povertyLevel =
+    povertyLevel =
             Config.getAsDouble("generate.demographics.socioeconomic.income.poverty", 11000);
```

Comment:
You changed this default value from `11000` to `12880` in `synthea.properties` so if the `12880` was intentional you may want to make this consistent.